scripts/DllManipulator.cs

```diff
@@ -17,6 +17,18 @@ public partial class DllManipulator
         public const string DLL_PATH_PATTERN_ASSETS_MACRO = "{assets}";
         public const string DLL_PATH_PATTERN_PROJECT_MACRO = "{proj}";
         private const string CRASH_FILE_NAME_PREFIX = "unityNativeCrash_";
+        public static readonly string[] DEFAULT_ASSEMBLY_NAMES = {"Assembly-CSharp"
+            #if UNITY_EDITOR
+            , "Assembly-CSharp-Editor"
+            #endif
+        };
+        public static readonly string[] INTERNAL_ASSEMBLY_NAMES = {"MCpiroman.UnityNativeTool"
```

Comment:
don't know if it matters, but my nick is not spelled in the same casing (if it does I prefer lowercase)